tests/test_retype.py

```diff
@@ -53,30 +53,33 @@ def test_can_run_against_current_directory(tmp_path):
 class RetypeTestCase(TestCase):
     maxDiff = None
 
-    def assertReapply(
```

Comment:
I accidentally had too much whitespace in my expected output, but I was stepping through what the assertion was doing to figure out if I should be using a different one and I realized they were all duplicates some of which having slightly different orders so I figured I'd remove that duplication.